python/pyspark/sql/functions.py

```diff
@@ -1921,10 +1921,12 @@ def from_json(col, schema, options={}):
 @since(2.1)
 def to_json(col, options={}):
     """
-    Converts a column containing a [[StructType]] or [[ArrayType]] of [[StructType]]s into a
-    JSON string. Throws an exception, in the case of an unsupported type.
+    Converts a column containing a :class:`StructType`, :class:`ArrayType` of :class:`StructType`s,
```

Comment:
I believe `` :class:`StructType`s `` should be ``:class:`StructType`\\s``. Currently, this is rendered as below:

<img width="215" alt="2017-09-14 7 12 52" src="https://user-images.githubusercontent.com/6477701/30424466-05e7532a-9981-11e7-8b37-d83268495cc0.png">

Reply:
ok, I'll modified it.  Because I'm not really familiar with python, thanks for your suggestions. :)

Reply:
That's fine :).